esmvalcore/_task.py

```diff
@@ -625,12 +629,12 @@ def _collect_provenance(self):
 
     def __str__(self):
         """Get human readable description."""
-        txt = "{}:\nscript: {}\n{}\nsettings:\n{}\n".format(
-            self.__class__.__name__,
-            self.script,
-            pprint.pformat(self.settings, indent=2),
-            super(DiagnosticTask, self).str(),
-        )
+        settings_string = pprint.pformat(self.settings, indent=2)
+        txt = (f"{self.__class__.__name__}:\n"
+               f"script: {self.script}\n"
+               f"settings:\n{settings_string}\n"
+               f"{super(DiagnosticTask, self)}\n")
```

Comment:
Any reason why you use arguments to super() here?

Reply:
This was already there, I don't know why so I left it.

Reply:
@stefsmeets promises to fix this  in followup PR ;-)

Reply:
> Any reason why you use arguments to super() here?

It's Python 2 syntax